### PR TITLE
feature(endpoint) delete gif endpoint [Start #188]

### PR DESCRIPTION
**Pull request by beblicarl**
#### Pre-Submission Checklist

-   [x] Your pull request targets the `main` branch.
-   [x] Branch name is descriptive and follows the guidelines [here](https://github.com/andela/bestpractices/wiki/Git-naming-conventions-and-best-practices).
-   [x] All tests pass . Use `git commit --amend` to amend any fixes.

#### Checklist:

-   [x] All tests are passing.
-   [x] Tested changes locally.
-   [x] Improved documentation or readme where necessary.
-   [x] Your commit messages format follows our commit message guideline [here](https://github.com/andela/bestpractices/wiki/Git-naming-conventions-and-best-practices#commits).

#### What does this PR do?

This PR deletes a gif by id
#### How should this be manually tested?

1. Start the server with `npm run dev
`2. Make  DELETE request to this endpoint `http://localhost:3000/api/v1/gifs/<gifid>
`3. This successfully returns `GIF image has been successfully deleted`

#### What are the relevant issues?
We needed to have an endpoint for deleting gifs

### CODE REVIEW

File: test/utils.js [hunk 1]
```diff
@@ -17,7 +20,63 @@ const setupDB = async() => {
...
+			`, [newData.firstName , newData.lastName , newData.email, 
+				passwordHash, newData.gender, newData.role ,
+				newData.department, newData.address, newData.jobRole,
+				newData.refreshToken
+			]
+		)
+		return newUser.rows[0]
+		
+	},
+	async insertPost(user,overrides = {}){
```
Comment (jocrah) [inline test/utils.js:55]: @beblicarl the overrides is just fine for this. all you have to do is pass it `userId`

File: src/services/posts/delete-post.test.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+
+describe('DELETE gif', () => {
+	let user
+	before(async ()=>{
+		 user = await fixtures.insertUser()  
+	})
+	it('should delete a gif', async () => {
+		
+	    const post = await fixtures.insertPost({userId : user.id})
```
Comment (jocrah) [inline src/services/posts/delete-post.test.js:14]: pass the `type` also here else the test will not be really accurate

Comment (beblicarl) [reply src/services/posts/delete-post.test.js:14]: okayy but the type will be hardcoded to gif this time around right?

Comment (jocrah) [reply src/services/posts/delete-post.test.js:14]: yep

**Review by jocrah — APPROVED** [review]

File: src/services/posts/delete-post.test.js (hunk 2 above)
Comment (beblicarl) [reply src/services/posts/delete-post.test.js]: done and pushed

**Pull request merged**